lte/gateway/c/core/oai/tasks/mme_app/mme_events.cpp

```diff
@@ -56,15 +54,15 @@ void event_client_init(void) { init_eventd_client(); }
  * @param event_tag
  * @return response code
  */
-static int report_event(folly::dynamic& event_value,
+static int report_event(nlohmann::json& event_value,
```

Comment:
<3 So happy this got done :). How do you feel about splitting the JSON changes into a separate PR? I think it'll be easier to review as this set of changes are very straightforward.

Reply:
Done, the new PR for the JSON changes is now  #11704.